client/basic/basic.go

```diff
@@ -91,3 +101,93 @@ func (c *client) PostTx(hexTx []byte, param map[string]string) ([]tx.TxCommitRes
 
 	return txResult, nil
 }
+
+func (c *client) WsGet(path string, constructMsg func([]byte) (interface{}, error), closeCh chan struct{}) (<-chan interface{}, error) {
+	u := url.URL{Scheme: types.DefaultWSSchema, Host: c.baseUrl, Path: fmt.Sprintf("%s/%s", types.DefaultWSPrefix, path)}
+	conn, _, err := websocket.DefaultDialer.Dial(u.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+	conn.SetPingHandler(nil)
+	conn.SetPongHandler(
+		func(string) error {
+			conn.SetReadDeadline(time.Now().Add(MaxReadWaitTime))
+			return nil
+		})
+	messages := make(chan interface{}, 0)
+	finish := make(chan struct{}, 0)
+	keepAliveCh := time.NewTicker(30 * time.Minute)
+	pingTicker := time.NewTicker(10 * time.Second)
+	go func() {
```

Comment:
since `err := conn.ReadJSON(&response)` is a block call, start a goroutine to response user's close action,